TimeOfFlight/include/TOFEstimators.h

```diff
@@ -3,22 +3,36 @@
 
 #include "marlin/Processor.h"
 #include "lcio.h"
+
+//std stuff
+#include <iostream>
 #include <string>
 #include <vector>
+#include <random>
+#include <functional>
+#include <memory>
+using std::string, std::vector, std::endl;
```

Comment:
Can we have these at least inside the class definition? Or even better use them in their full glory in headers and maybe only try to shorten them in .cc files when actually necessary. Using things from the `std` namespace this way can lead to interesting things in such a large scope.